src/pydata_sphinx_theme/assets/styles/sections/_skip-link.scss

```diff
@@ -12,18 +12,24 @@
   right: 0;
   text-align: center;
   background-color: var(--pst-color-warning);
-  // Ensure we are using a WCAG conformant colour
-  color: var(--pst-color-warning-text) !important;
   padding: 0.5rem;
   z-index: $zindex-modal;
   border-bottom: 1px solid var(--pst-color-border);
 
   // This shows / hides the button
   transform: translateY(-100%);
   transition: transform 150ms ease-in-out;
-  &:focus {
+  &:focus-within {
     transform: translateY(0%);
-    // ensure this is visible
-    outline: 3px solid $foundation-black;
+  }
+
+  a {
+    // Ensure we are using a WCAG conformant colour
+    color: var(--pst-color-warning-text) !important;
+
+    &:focus-visible {
+      // use color with sufficient contrast
+      outline-color: $foundation-black;
```

Comment:
@smeragoel I'll need guidance here on what color to set the focus ring for the skip link.

Reply:
We talked on Slack and decided on this tone of black for now